src/ert/scheduler/lsf_driver.py

```diff
@@ -444,6 +449,24 @@ async def _get_exit_code_from_bhist(self, job_id: str) -> int:
 
         return LSF_FAILED_JOB
 
+    async def _dump_bhist_job_summary_to_runpath(self, job_id: str) -> None:
+        runpath = self._jobs[job_id].runpath
+        job_name = self._jobs[job_id].name
+        file_path = runpath / f"{job_name}.LSF-bhist-l"
+        bhist_with_args: list[str] = [
+            str(self._bhist_cmd),
+            "-l",  # long format
+            str(job_id),
+        ]
+        process_success, process_message = await self._execute_with_retry(
+            bhist_with_args,
+            retry_codes=(FLAKY_SSH_RETURNCODE,),
+            retries=3,
+            retry_interval=self._sleep_time_between_cmd_retries,
+        )
+        if process_success:
+            Path(file_path).write_text(process_message)
```

Comment:
Now I wonder what's the better approach as I try to do something similar here: https://github.com/equinor/ert/pull/7836

Reply:
I think we should have both